<launch>
	<!-- Lidar odometry param -->
	<rosparam file="$(find lvi_sam)/config/kitti_custom/2011_09_30/KITTI_lidar.yaml" command="load" />
	<!-- VINS config -->
  <rosparam file="$(find lvi_sam)/config/kitti_custom/2011_09_30/KITTI_camera.yaml" command="load" />
  
  <!-- path for each camera intrinsics -->
  <param name="cam_configs_dir" type="string" value="$(find lvi_sam)/config/kitti_custom/2011_09_30/cams" />

    <!--- Robot State TF -->
    <!-- <include file="$(find lvi_sam)/launch/include/module_robot_state_publisher.launch" /> -->
    <!--- SAM -->
    <include file="$(find lvi_sam)/launch/include/module_sam.launch" />
</launch>
